examples/SmartShop/contracts/SellerContract.aes

```diff
@@ -0,0 +1,36 @@
+contract Seller =
+  record state = {seller_key : address, buyer_contract: address, status : string, price : int}
+  
+  public stateful function init(bc : address, price : int) : state =
+    {seller_key = Call.caller, buyer_contract = bc, status = "undefined", price = price }
+    
+  public stateful function send_item() : bool =
+    if(check_money())
```

Comment:
maybe when buyer bought a product/item you should change the owner of product/item

Reply:
Same as comment above.